source/howto/CreateVMFromCommandLine.rst

```diff
@@ -152,11 +114,10 @@ For example:-
 
 .. code-block:: bash
 
-  openstack --debug server create --flavor m1.tiny --image Ubuntu-Xenial-NoGui --nic net-id=Internal --security-group default --key-name xbe91637 test_2018-10-30_1357
-
+  openstack --debug server create --flavor <flavor ID/name> --image <image ID/name> --network <network> --security-group default --key-name <key name> <server name>
 ##########
 References
 ##########
```

Comment:
This warning is because you need another line between the code-block and the new section